### PR TITLE
Added data object and resource files for Score database model

### LINKED ISSUE #4: Put retrieved form data from API request to store into database

### PR DESCRIPTION
**Pull request by joannetsaii**
Resolved #4

### CODE REVIEW

File: docs-web/src/main/java/com/sismics/docs/rest/resource/ScoreResource.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        skills = ValidationUtil.validateLength(skills, "skills", 1, 1, false);
+        experience = ValidationUtil.validateLength(experience, "experience", 1, 1, false);
+        transcriptGPA = ValidationUtil.validateLength(transcriptGPA, "transcript_gpaa", 1, 1, false);
+        match = ValidationUtil.validateLength(match, "match", 1, 1, false);
+        Date createDate = ValidationUtil.validateDate(createDateStr, "create_date", true);
+
+        // Create the score
+        Score score = new Score();
+        score.setDocumentId(documentId);
+        score.setUserId(principal.getId());
```
Comment (claralam) [inline docs-web/src/main/java/com/sismics/docs/rest/resource/ScoreResource.java:75]: Could you explain what principal is? I'm a little confused and I'm not sure if I see any other references to it.

Comment (joannetsaii) [reply docs-web/src/main/java/com/sismics/docs/rest/resource/ScoreResource.java:75]: From my understanding, a principal represents the user identity and is used to determine the rights to access objects in a system.

Comment (claralam) [reply docs-web/src/main/java/com/sismics/docs/rest/resource/ScoreResource.java:75]: Okay, thanks for the explanation!

**Review by claralam — APPROVED** [review]
LGTM!

**Pull request merged**